libraries/standard/json/src/json.c

```diff
@@ -481,6 +511,8 @@ static bool_ strnEq( const char * a,
 {
     size_t i;
 
+    assert( ( a != NULL ) && ( b != NULL ) );
```

Comment:
`assert(__CPROVER_r_ok(a,n) && __CPROVER_r_ok(b,n))` [Or possibly `n+1`, if we have a null terminator]

Reply:
I'd prefer to keep the proof separate.